docs/contributing.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+Open blog is before all an open source project to share a publication of blog posts with the community.
+
+> Developed by the community for everyone freely
+
+## Share your blog posts :rocket:
+
+Before adding your blog post to Open blog, you should check those 3 steps.
+
+### Post rules
+
+- Post should follow the [Code of Conduct](https://github.com/open-blog-initiative/open-blog.dev/blob/master/CODE_OF_CONDUCT.md).
+- Post can't be direct advertising.
+- Post can't break any copyright in its content.
+- Post should not be published in another blog.
+- Post should be written in one of those languages:
+  - French :fr:
+  - English :gb:
+
+NB: For now we only support two languages because those languages are maintainer ones.
+If you want to be part of the maintainer of Open Blog for another language, just contact one of the maintainer.
+
+### How to :question:
+
+#### Install
+
+In order to work locally on your blog post and see how it will be displayed when merged, be sur to have NodeJS installed (>=10.15).
+
+- Fork this repository ["Fork me !"](https://github.com/open-blog-initiative/open-blog.dev)
+- Clone your forked version
+- Install dependencies
+
+```sh
+npm install
+```
+
+or
+
+```sh
+yarn install
+```
+
+- Start dev server
+
+```sh
+npm run docs:dev
+```
+
+or
+
+```sh
+yarn docs:dev
+```
+
+#### Create your post file
+
+In `docs/post` folder, create a folder with your github name.
```

Comment:
On devrait peut être faire un super simple cli pour générer un nouvel article?

Reply:
Yep, if you want, go for it 💡 or open a issue at least 😉

Reply:
I'll take a look on friday :)